seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/java/org/apache/seatunnel/e2e/connector/kafka/KafkaIT.java

```diff
@@ -294,17 +296,100 @@ public void testSourceKafkaStartConfig(TestContainer container)
     }
 
     @TestTemplate
-    @DisabledOnContainer(TestContainerId.SPARK_2_4)
+    @DisabledOnContainer(value = {TestContainerId.FLINK_1_13, TestContainerId.SPARK_2_4})
```

Comment:
**TestContainerId.FLINK_1_13**
Why is it banned?

Reply:
reverted